rbac/management/tenant_service/v2.py

```diff
@@ -219,7 +219,9 @@ def _disable_user_in_tenant(self, user: User):
         try:
             mapping = TenantMapping.objects.filter(tenant__org_id=user.org_id).get()
             default_group_uuid = str(mapping.default_group_uuid)  # type: ignore
+            default_admin_group_uuid = str(mapping.default_admin_group_uuid)  # type: ignore
             tuples_to_remove.append(Group.relationship_to_user_id_for_group(default_group_uuid, user_id))
+            tuples_to_remove.append(Group.relationship_to_user_id_for_group(default_admin_group_uuid, user_id))
```

Comment:
Are we doing this also if user is not admin ? (Maybe it is safer to do it for both.)

Reply:
Right, that's the plan, we are doing this for even users not admin.